### PR TITLE
Remark command change in DG

### PR DESCRIPTION
**Pull request by kangqiao322**

### CODE REVIEW

**Comment by jetlfj** [comment]
Remember to update the .png files too, if not the diagrams will still be incorrect.

**Comment by jetlfj** [comment]
The images are in the wrong directory, they should be in `/images/`, not `/diagrams/`.

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -346,11 +346,11 @@ Step 4. If the format is correct, `RemarkCommandParser` will create a `RemarkCom
 
 Step 5. The `LogicManager` executes the `RemarkCommand`.
 
-Step 6. The `RemarkCommand` obtains a list of `Internship`s via the `getFilteredInternshipList()` method
-which is exposed to the `Model` interface as `Model#getFilteredInternshipList()`
+Step 6. The `RemarkCommand` obtains a list of `Internship`s via the `getSelectedInternshipList()` method
+which is exposed to the `Model` interface as `Model#getSelectedInternshipList()`
```
Comment (chantellyu) [inline docs/DeveloperGuide.md:350]: Maybe you could include a step where if nothing is selected, an error will be thrown here for more detail!

**Review by chantellyu — COMMENTED** [review]
Your RemarkSequenceDiagram is a little outdated! It still has the INDEX field in the command.

**Pull request merged**